javalib/src/main/scala/java/io/InputStream.scala

```diff
@@ -136,17 +134,13 @@ abstract class InputStream extends Closeable {
     }
 
     def readLargeN(len: Int): Array[Byte] = {
-      /* The byteStore is not expected to be accessed concurrently.
-       * ConcurrentedLinkedDeque is used here because the Scala Native JSR-166
-       * code is newer, more studied, and likely to execute faster
-       * than the SN LinkedListDequeue implementation. FUD, not measurement.
-       *
-       * Using a Deque rather than, say, a ByteArrayOutputStream may briefly
+      /*
+       * Using, say, a ByteArrayOutputStream may briefly
```

Comment:
The basic concern of the chosen data structure briefly
exceeding the JVM description is, I believe, still valid.

I think one can get to 2*nbytes + (8 * ArrayList.size()).

The actual text appears to be superseded and is now
time consuming word salad.